lab_1/main.py

```diff
@@ -36,9 +71,21 @@ def calculate_frequencies(tokens: list) -> dict:
     e.g. tokens = ['weather', 'sunny', 'man', 'happy']
     --> {'weather': 1, 'sunny': 1, 'man': 1, 'happy': 1}
     """
-    pass
+    frequency = {}
+    if type(tokens) == list:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.